Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx

```diff
@@ -95,6 +95,10 @@ vtkMRMLCameraWidget::vtkMRMLCameraWidget()
   this->SetKeyboardEventTranslation(WidgetStateIdle, vtkEvent::NoModifier, 0, 0, "plus", WidgetEventCameraZoomIn);
   this->SetKeyboardEventTranslation(WidgetStateIdle, vtkEvent::NoModifier, 0, 0, "minus", WidgetEventCameraZoomOut);
 
+  // Toggle stabilizer
+
+  this->SetKeyboardEventTranslation(WidgetStateIdle, vtkEvent::NoModifier, 0, 0, "a", WidgetEventToggleStabilizer);
```

Comment:
`a` keyboard shortcut without modifier is already used in several extensions and this may disrupt those. For example, for moving devices in SlicerHeart we use the WASD and QE keys (with Shift for fine adjustment). It would be also too easy to accidentally hit the `a` key and then users would have no idea why their 3D view got permanently broken (until an application restart).

It would be better to use a shortcut with a modifier, something that is close to the cursor arrow keys (which rotate the 3D view). For example, Ctrl+`,` or Ctrl+`.` or Ctrl+`/`.